api/types/trust.go

```diff
@@ -57,9 +59,19 @@ var CertAuthTypes = []CertAuthType{HostCA, UserCA, DatabaseCA, OpenSSHCA, JWTSig
 // major version, so that we can avoid erroring out when a potentially older
 // remote server doesn't know about them.
 func (c CertAuthType) NewlyAdded() bool {
+	return c.addedInVer().Major == semver.New(teleport.Version).Major
+}
+
+// addedInVer return version in which given CA was added.
+func (c CertAuthType) addedInVer() *semver.Version {
 	switch c {
+	case DatabaseCA:
+		return semver.New("9.0.0")
```

Comment:
```suggestion
		return &semver.Version{Major: 9}
```

Reply:
We could actually just return a `semver.Version` here rather than a pointer I guess.